cpp/harvest/nacjd.cc

```diff
@@ -452,6 +444,170 @@ void convertJSONtoMARC(int argc, char **argv) {
     LOG_INFO("Finished.");
 }
 
+void getStatistics(int argc, char **argv) {
+    // Parse args
+    if (argc != 3)
+        Usage();
+
+    std::string json_path = argv[2];
```

Comment:
This value will never change. Please use the const keyword and change it to constructor notation:
`const std::string json_path(argv[2]);`